codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -86,30 +86,43 @@ As one dynamic contract register catches up to the fetched blocknumebr of the ne
 merge itself into the next register and combine queries/addresses and queues until fully caught
 up to the root. 
 */
-type fetchStateData = {
+type registerData = {
   latestFetchedBlock: blockNumberAndTimestamp,
   contractAddressMapping: ContractAddressingMap.mapping,
   //Events ordered from latest to earliest
   fetchedEventQueue: array<Types.eventBatchQueueItem>,
   //Used to prune dynamic contract registrations in the event
   //of a rollback.
   dynamicContracts: DynamicContractsMap.t,
-  isFetchingAtHead: bool,
```

Comment:
Removed isFetchingAtHead since not every register needs to have it's own value for this.